.github/workflows/flowzone.yml

```diff
@@ -11,4 +11,7 @@ jobs:
   flowzone:
     name: Flowzone
     uses: product-os/flowzone/.github/workflows/flowzone.yml@master
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
+    with:
+      cloudflare_website: "balenaCloud-docs"
+      custom_website_build: "npm run build"
```

Comment:
My concern with this commit is that the option `custom_website_build` was never merged in Flowzone master, and this line is changed in a future commit. So this commit without the future commit is invalid and difficult to follow the blame.

Reply:
I agree on that. Let me go back and restore this change as well. My other commit includes the correct change.

Reply:
Resolved, how about now. Fixed that change in the second commit.

Reply:
also wrote a blog post: https://mixstersite.wordpress.com/2023/01/02/edit-git-commits/